zulip_bots/zulip_bots/run.py

```diff
@@ -3,12 +3,33 @@
 from __future__ import absolute_import
 
 import logging
-import optparse
+import argparse
 import sys
+import os
 from types import ModuleType
 from importlib import import_module
+from os.path import basename, splitext
+
+import six
 
 from zulip_bots.lib import run_message_handler_for_bot
+from zulip_bots.provision import provision_bot
+
+
+def import_module_from_source(path, name=None):
+    if name is None:
+        name = splitext(basename(path))[0]
+
+    if six.PY2:
```

Comment:
I'm not sure actually, I started a discussion in `#integrations` once but it didn't lead to any firm conclusions on this matter! I feel like I remember Tim mentioning that we definitely want the `zulip` API package to support Python 2. And in a past discussion, someone mentioned that we want to move the bots API to the `zulip` API package as well. If we end up doing that, then this code will have to support Python 2. I think this definitely warrants a longer discussion though.